packages/botonic-react/src/webchat/replies.jsx

```diff
@@ -49,11 +49,9 @@ export const WebchatReplies = props => {
       autoHide={scrollbarOptions.autoHide}
     >
       <RepliesContainer
+        className='replies-container'
         justify={justifyContent}
         wrap={flexWrap}
-        style={{
-          ...props.style,
-        }}
```

Comment:
is this not used by any user?

Reply:
these props.style are never passed to the WebchatReplies component. You can see it in the other modified file in this PR